dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -8006,6 +8019,7 @@ CONFIG_F2FS_FS_LZ4HC=y
 CONFIG_F2FS_FS_ZSTD=y
 # CONFIG_F2FS_IOSTAT is not set
 # CONFIG_F2FS_UNFAIR_RWSEM is not set
+# CONFIG_BCACHEFS_FS is not set
```

Comment:
Looks like something we should enable and either add descriptions or add backlog to #533